.github/workflows/test-suite.yml

```
@@ -10,33 +10,39 @@ on:
 
 jobs:
   tests:
-    name: "Tests"
-    runs-on: ${{ matrix.operating-system }}
+    name: "Tests ${{ matrix.php-version }} ${{ matrix.dependencies }} (Symfony ${{ matrix.symfony-version }})"
+    runs-on: "ubuntu-latest"
     strategy:
+      fail-fast: false
       matrix:
         dependencies:
-          - "locked"
           - "lowest"
           - "highest"
         php-version:
-          - "8.1"
-          - "8.2"
           - "8.3"
           - "8.4"
-        operating-system:
-          - "ubuntu-latest"
+          - "8.5"
+        symfony-version:
+          - "5.4.*"
+          - "6.4.*"
+          - "7.3.*"
+          - "7.4.*"
+          - "8.0.*"
+        exclude:
+          - php-version: 8.3
+            symfony-version: "8.0.*"
```

@norberttech with the following matrix config it can be reduced to only 5 CI jobs
```suggestion
      matrix:
        include:
          - php-version: 8.3
            symfony-version: "5.4.*"
            dependencies: "lowest"
          - php-version: 8.3
            symfony-version: "6.4.*"
            dependencies: "highest"
          - php-version: 8.4
            symfony-version: "7.3.*"
            dependencies: "highest"
          - php-version: 8.4
            symfony-version: "7.4.*"
            dependencies: "highest"
          - php-version: 8.5
            symfony-version: "8.0.*"
            dependencies: "highest"
```

---

Maybe a run with `dependencies: "locked"` is missing

---

lets keep it as is for now, if it become too long/complicated we can reduce it to what you suggested above, thanks for the example!